### ISSUE TITLE
Large filesize

### ISSUE THREAD

**Issue by bcvery1** [markreidvfx/pyaaf2#44]
I'm creating an AAF with a single embedded instance, which is a dnxhd file containing a single frame.  The dnxhd file is 184kb, created with the following command:
`ffmpeg -f lavfi -i testsrc=size=1920x1080 -v error -y -nostdin -an -c:v dnxhd -pix_fmt yuv422p -b:v 36M -framerate 24 -frames:v 1 sample.dnxhd`

I've then used a trimmed down version of the example "Writing an AAF" code:
```python
import aaf2

if __name__ == '__main__':
	with aaf2.open('out.aaf', 'w+') as f:
		master_mob = f.create.MasterMob()
		master_mob.import_dnxhd_essence("sample.dnxhd", 24)
		f.content.mobs.append(master_mob)
```

The resultant AAF is 12Mb.  Is there anything I can do to reduce this size?  Please note that I require to embed the media and cannot keep it external.

**Comment by markreidvfx** [comment]
yup thats not good! Looks like you found a regression, should be fix now in 22b7b87ac8192dfff3d340dd7937084b191269e9

**Comment by bcvery1** [comment]
Perfect - that change has fixed this issue
